### PR TITLE
Set Logistics Requisition state to Done when all lines are quoted or can...

### PR DESCRIPTION
**Pull request by yvaucher**
...celled

and if at least one line is quoted, otherwise if all line are cancelled, we set LR as cancelled

### CODE REVIEW

File: logistic_requisition/model/logistic_requisition.py [hunk 1]
```diff
@@ -491,6 +501,9 @@ def _do_cancel(self):
         vals = {'state': 'cancel',
                 'logistic_user_id': False}
         self.write(vals)
+        # When all lines of a requisition are 'quoted' or 'cancel, it should be
```
Comment (foutoucour) [inline logistic_requisition/model/logistic_requisition.py]: s/'cancel/'cancel'
But nothing to worry really

**Comment by foutoucour** [comment]
:+1:

**Comment by lepistone** [comment]
:+1: thanks

File: logistic_requisition/model/logistic_requisition.py (hunk 1 above)
Comment (yvaucher) [inline logistic_requisition/model/logistic_requisition.py]: Fixed thanks

**Comment by guewen** [comment]
One flake8 error:

```
./logistic_requisition/model/logistic_requisition.py:506:80: E501 line too long (80 > 79 characters)
```

**Pull request merged**